.travis.yml

```diff
@@ -1,14 +1,19 @@
-language: python
-python:
-    - "3.6"
-
-sudo: false
-
-before_install:
-    - cd python
-
-install:
-    - pip install -r requirements.txt
-
-script:
-    - python -m pytest
+matrix:
```

Comment:
This is sick. Can you explain how this works? I never understood the matrix idea.

Reply:
Honestly I don't really understand how it works either haha I just looked at some examples online. It seems like you can run multiple 'instances' independently of one another making it easy to write separate CI pipelines for different languages.